tests/openssh.rs

```diff
@@ -17,6 +21,16 @@ async fn it_connects() {
     session.close().await.unwrap();
 }
 
+#[tokio::test]
+#[cfg_attr(not(ci), ignore)]
+async fn control_dir() {
+    let session = SessionBuilder::default().control_directory(&path()).connect(&addr()).await.unwrap();
```

Comment:
I think you need to run `rustfmt` on this file — that's what CI is complaining about :)

Reply:
Not sure it's related to my changes -- I'm seeing lots of this error in that file:
```
error[E0670]: `async fn` is not permitted in the 2015 edition
  --> /home/lincolnb/rust/openssh-rs/tests/openssh.rs:14:1
   |
14 | async fn it_connects() {
   | ^^^^^ to use `async fn`, switch to Rust 2018
   |
   = help: set `edition = "2018"` in `Cargo.toml`
   = note: for more on editions, read https://doc.rust-lang.org/edition-guide
```

Reply:
That's weird... Do you have a very old version of Rust installed/

Reply:
No, turns out you have to specify --edition 2018 on the command line!